libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -209,7 +225,8 @@ void RemoteProcessorGroupPort::onTrigger(const std::shared_ptr<core::ProcessCont
   std::string value;
 
   logger_->log_trace("On trigger %s", getUUIDStr());
-  if (url_.empty()) {
+  /*
```

Comment:
looks like this is handled elsewhere with your changes.  I can clean this up on merge if that works for you.

Reply:
that's fine by me